x-pack/plugins/security_solution/common/endpoint/types/authz.ts

```diff
@@ -22,6 +22,8 @@ export interface EndpointAuthz {
   canReadFleetAgents: boolean;
   /** If the user has permissions to write Fleet Agents */
   canWriteFleetAgents: boolean;
+  /** If the user has permissions to write Integration policies */
+  canWriteIntegrationPolicies: boolean;
```

Comment:
Maybe clarify the JSDOC comment to say "from the Fleet add integration page"??

Reply:
added ee762ef16b18135d59f36b4cbdd534200ff4ea90